workflow/automation/org/kisti/srf_gen.pbs

```diff
@@ -33,15 +33,21 @@ echo $start_time
 
 python $gmsim/workflow/workflow/automation/execution_scripts/add_to_mgmt_queue.py $MGMT_DB_LOC/mgmt_db_queue $REL_NAME SRF_GEN running $SLURM_JOB_ID --start_time "$start_time" --nodes $SLURM_NNODES --cores $SLURM_CPUS_PER_TASK --wct 01:00:00
 
-echo python $gmsim/Pre-processing/srf_generation/input_file_generation/realisation_to_srf.py $REL_YAML
-python $gmsim/Pre-processing/srf_generation/input_file_generation/realisation_to_srf.py $REL_YAML
+if [ "${REL_FILEPATH##*.}" == "csv" ]; then 
+    echo python $gmsim/Pre-processing/srf_generation/input_file_generation/realisation_to_srf.py $REL_FILEPATH
+    python $gmsim/Pre-processing/srf_generation/input_file_generation/realisation_to_srf.py $REL_FILEPATH
+else
+    echo python $gmsim/Pre-processing/srf_generation/input_file_generation/generate_type5_srf.py $REL_FILEPATH $SRF_DIR
+    python $gmsim/Pre-processing/srf_generation/input_file_generation/generate_type5_srf.py $REL_FILEPATH $SRF_DIR
+fi
+
 
 end_time=`date +$runtime_fmt`
 echo $end_time
 
-INFO_PATH=${REL_YAML%.*}.info
-STOCH_PATH=${REL_YAML%.*}.stoch
-SIM_PARAMS_PATH=${REL_YAML%.*}.yaml
+INFO_PATH=${REL_FILEPATH%.*}.info
+STOCH_PATH=${REL_FILEPATH%.*}.stoch
+SIM_PARAMS_PATH=${REL_FILEPATH%.*}.yaml
```

Comment:
If REL_FILEPATH is already yaml, what happens here?

Reply:
Problems! We'll have to talk about what to do with these files in a way that doesn't completely ruin everything. If we change the sim params path here we'll need to change everything downstream...

Reply:
Actually, reading this file I am not even sure what `SIM_PARAMS` even is?